### PR TITLE
Use regex test method in array filter method

### PR DESCRIPTION
**Pull request by geryogam**

### CODE REVIEW

File: files/en-us/web/javascript/guide/regular_expressions/assertions/index.md [hunk 1]
```diff
@@ -187,19 +187,19 @@ tangs on ihe greon traa`;
 
 // 1) Use ^ to fix the matching at the beginning of the string, and right after newline.
 buggyMultiline = buggyMultiline.replace(/^t/gim, "h");
-console.log(1, buggyMultiline); // fix 'tey', 'tangs' => 'hey', 'hangs'. Avoid 'traa'.
+console.log(1, buggyMultiline); // fix 'tey' => 'hey' and 'tangs' => 'hangs' but do not touch 'traa'.
 
 // 2) Use $ to fix matching at the end of the text.
 buggyMultiline = buggyMultiline.replace(/aa$/gim, "ee.");
-console.log(2, buggyMultiline); // fix  'traa' => 'tree'.
+console.log(2, buggyMultiline); // fix 'traa' => 'tree'.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/regular_expressions/assertions/index.md:194]:
```suggestion
console.log(2, buggyMultiline); // fix 'traa' => 'tree.'.
```

Comment (geryogam) [reply files/en-us/web/javascript/guide/regular_expressions/assertions/index.md]: Updated.

**Review by Josh-Cena — APPROVED** [review]
Thank you!

**Comment by geryogam** [comment]
Thanks!

**Pull request merged**